uxsdcxx/cpp.py

```diff
@@ -182,13 +218,29 @@ def _gen_load_simple(t: UxsdSimple, input: str) -> str:
 	else:
 		return "load_%s(%s)" % (utils.to_snakecase(t.cpp), input)
 
-def _gen_load_element_complex(t: UxsdElement, parent: str) -> str:
+def _gen_load_element_complex(t: UxsdElement, parent: str, complex_type_map : Dict[str, UxsdComplex]) -> str:
 	assert isinstance(t.type, UxsdComplex)
-	out = ""
+	out = "{\n"
+
+	args = ["data"]
+	load_args = []
+	
+	for attr in complex_type_map[t.type.name].attrs:
```

Comment:
Why not just `t.type.attrs`?

Reply:
Is that the same as what I did?

Reply:
Yes, `t.type` gives the exact same `UxsdComplex` you get from the map.